include/graph.h

```diff
@@ -101,8 +101,9 @@ class Graph {
    */
   void write_binary(const std::string &filename);
 
-  std::chrono::steady_clock::time_point flush_call;
-  std::chrono::steady_clock::time_point flush_return;
+  // time hooks for experiments
+  std::chrono::steady_clock::time_point flush_start;

```

Comment:
Why did we rename these

Reply:
for consistency

Reply:
Okay

Reply:
Okay

Reply:
Can you put in a pull request for the experiment repo that changes the query_expr in main to use these time hooks?

Reply:
Yeah, it’s in draft rn